nipap/nipap/xmlrpc.py

```diff
@@ -24,8 +24,7 @@
 from authlib import AuthFactory, AuthError
 
 
-def setup():
-    app = Flask('nipap.xmlrpc')
+def setup(app):
     Compress(app)
```

Comment:
Put this in nipapd, where app is created.

Reply:
Done in new commit.